doc/manpages/dinitctl.8.m4

```diff
@@ -298,6 +301,18 @@ The value can be provided on the command line or retrieved from the environment
 called in.
 Any subsequently started or restarted service will have these environment variables available.
 This is particularly useful for user services that need access to session information.
+.TP
+\fBsignal\fR
+Send a signal to a specified service.
+The \fIsignal-number\fR can either be specified as integer or signal name.
+Signal names are only defined for these signals:
+\fBHUP\fR, \fBKILL\fR, \fBUSR1\fR, \fBUSR2\fR, \fBSTOP\fR, \fBINT\fR, \fBTERM\fR,
+\fBCONT\fR, \fBQUIT\fR and if system supports it; \fBINFO\fR.
+Other system specific signals, like Linux real-time signals, have to be specified as integer.
+Dinit will accept any signal which supported by \fBkill(2)\fR syscall.
+.sp
+The \fB--list/\-l\fR option can be used (instead of \fIsignal-number\fR and \fIservice-name\fR)
```

Comment:
"--list/-l" will all be in bold, it's probably better to say "--list or -l" or at least make the slash regular font instead of bold. I will fix it later.